server/__tests__/app.test.js

```diff
@@ -114,3 +113,58 @@ afterAll(() => {
         })
     })
   })
+
+  describe('5. GET /api/reviews/:review_id/comments', () =>{
+
+    test("status:200, returns an array of comments matching the parametric review_id", ()=>{
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200)
+        .then((response)=>{
+            const comments = response.body.comments;
+            expect(true).toBe(Array.isArray(comments));
+            expect(comments).toHaveLength(3);
+            comments.forEach(comment =>{
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        review_id: expect.any(Number),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number)
+                    })
+                )
+            }) 
+            expect(comments).toBeSortedBy('created_at', {descending: true});
+        })
+    })
+
+    test("status:200, returns an empty array matching the parametric review_id", ()=>{
```

Comment:
I would add to the description that this is a case where the id is valid and exists but there happens to be no comments for it

Reply:
great rich cheers